gpu/segmentation/include/pcl/gpu/segmentation/impl/gpu_extract_clusters.hpp

```diff
@@ -96,13 +100,10 @@ pcl::gpu::extractEuclideanClusters (const pcl::PointCloud<pcl::PointXYZ>::Ptr  &
     // once the area stop growing, stop also iterating.
     do
     {
-      // Host buffer for results
-      std::vector<int> sizes, data;
```

Comment:
Should we be calling `{data,size}.clear()` somewhere? Or is the intention to persist their data (and memory) between the do-while loops?

Reply:
I think calling clear on the vector is a good idea as it shows the programmers intent clearly! I have now called it at the beginning of the for loop.